src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx

```diff
@@ -104,14 +125,7 @@ const ProjectsTable: React.FC = () => {
       maxWidth: 150,
       valueFormatter: (params) => params.value.length
     },
-    {
-      ...baseColDef,
-      field: 'status',
-      headerName: 'Status',
-      type: 'singleSelect',
-      valueOptions: Object.values(WbsElementStatus),
-      maxWidth: 100
-    }
+    statusColumn
   ];
 
   const filterValues = JSON.parse(
```

Comment:
I'm getting an undefined error on this line:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/66755a7a-5f7c-4ad1-b189-f574b4f91b25)

Reply:
huh thats weird because it looks like valid json and also it runs fine on my machine

Reply:
note for self later: 
<img width="489" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/5ca66a9b-2179-4aac-b703-38a99f76ce21">